lib/ansible/modules/database/mysql/mysql_db.py

```diff
@@ -120,6 +120,7 @@
 
 
 def db_exists(cursor, db):
+    db = db.replace("`", r"\`")
```

Comment:
I think it not needed.
The best is to do change `CREATE DATABASE %s` to `CREATE DATABASE IF NOT EXISTS %s`

Reply:
Wouldn't changing to "if not exists" break the check if the task made changes or not?